### DISCUSSION TITLE
LAB3 platform  can't find xilinx_u50_gen3x16_xdma_5_202210_1

### DISCUSSION THREAD

**Discussion by JENHAOLU** [bol-edu/HLS-SOC-Discussions#84] [Q&A]
我的平台找不到指定的 xilinx_u50_gen3x16_xdma_5_202210_1 
![Screenshot from 2023-10-04 04-53-17](https://github.com/bol-edu/HLS-SOC-Discussions/assets/112152469/c2eb0200-783f-4aa4-8614-6a10daddb694)
不知道有沒有人也是這樣，可以選別的來用嗎?

**Comment by kevinjantw** [comment]
> 我的平台找不到指定的 xilinx_u50_gen3x16_xdma_5_202210_1 
> 
> ![Screenshot from 2023-10-04 04-53-17](https://github.com/bol-edu/HLS-SOC-Discussions/assets/112152469/c2eb0200-783f-4aa4-8614-6a10daddb694)
> 
> 不知道有沒有人也是這樣，可以選別的來用嗎?
> 
> 

這是你的VM嗎？或是那裡的機器？

**Reply by JENHAOLU** [reply]
是VM的，我是使用助教給的安裝好Xilinx的包，不過這個u50似乎得自己下載，我晚點試試看，以下是找到的u50檔
AMD網址https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/alveo/u50.html

**Comment by boledulab** [comment]
助教的VM包裡沒有u50 安裝後也只能用來產生bitstream
https://www.xilinx.com/products/boards-and-kits/alveo/u50.html#gettingStarted
https://docs.xilinx.com/v/u/en-US/ug1370-u50-installation
驗證時會需要實體u50卡 你有這實驗需求? 可以到boledu slack私訊Kevin Jan

**Reply by JENHAOLU** [reply]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/112152469/40641077-b761-4e65-9472-10f1b561c563)

因為我看教學要這個board，我已經找到去官網找到檔案了，有問題會再提問，感謝!
